.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: node_js
 node_js:
   - '10'
-  - '8'
```

Comment:
You will also have to set `node` to `>=10` in `engines` in `package.json`.

Reply:
I don't understand what you mean. I thought this should be removed from expected